forms-flow-web/src/apiManager/services/formatterService.js

```diff
@@ -124,6 +124,7 @@ export const listProcess = (processes) => {
       return {
         label: process.name,
         value: process.key,
+        tenant:process.tenantKey
```

Comment:
For this; you may need to map the processtenant from form process mapper.

Reply:
👍